page-decision-listing.php

```diff
@@ -0,0 +1,347 @@
+                $tax_qry_ary = [];
+
+                // Add Decision Restriction
+                if (!empty($restrict_decisions) && $restrict_decisions != 'no') {
+                    if (taxonomy_exists($restrict_decisions)) {
+                        // Decisions will only display with the given term ids - from a set taxonomy
+                        $restrict_term_ids = get_post_meta(get_the_ID(), 'restrict_decisions_by_' . $restrict_decisions, true);
+
+                        if (!empty($restrict_term_ids)) {
+                            $tax_qry_ary[] = array(
+                                'taxonomy' => $restrict_decisions,
+                                'field' => 'term_id',
+                                'terms' => $restrict_term_ids
+                            );
+                        }
+                    }
+                }
+
+                // decision Type Filter
+                if ($restrict_decisions != "decision_referral_type") {
+                    if (!empty($selected_decision_referral_type_id)) {
+                        $tax_qry_ary[] = array(
+                            'taxonomy' => 'decision_referral_type',
+                            'field' => 'term_id',
+                            'terms' => $selected_decision_referral_type_id
+                        );
+                    }
+                }
+
+                // decision Category Filter
+                if ($restrict_decisions != "decision_offence") {
+                    if (!empty($selected_decision_offence_id)) {
+                        $tax_qry_ary[] = array(
+                            'taxonomy' => 'decision_offence',
+                            'field' => 'term_id',
+                            'terms' => $selected_decision_offence_id
+                        );
+                    }
+                }
+
+                // decision Location Filter
+                if ($restrict_decisions != "decision_process") {
+                    if (!empty($selected_decision_process_id)) {
+                        $tax_qry_ary[] = array(
+                            'taxonomy' => 'decision_process',
+                            'field' => 'term_id',
+                            'terms' => $selected_decision_process_id
+                        );
+                    }
+                }
+
+                if (!empty($tax_qry_ary)) {
+                    $decision_args['tax_query'] = $tax_qry_ary;
+                }
+
+                $decision_query = new WP_Query($decision_args);
+                $decision_referral_type_filter_activated = get_post_meta(get_the_ID(), 'decision_referral_type_filter_activated', true);
+
+                if ($decision_query->have_posts()) {
+                    if ($decision_query->found_posts > 1) {
+                        $doc_count_text = $decision_query->found_posts . ' decisions';
+                    } elseif ($decision_query->found_posts == 1) {
+                        $doc_count_text = '1 decision';
```

Comment:
Should be put this in the function that means it would pick up a translation?  
Is it something like `_e("xxx","hale")` - although I think that echoes it out.

Reply:
Yes, probably a good idea.